src/main/cpp/Constants.cpp

```diff
@@ -0,0 +1,42 @@
+#include "Constants.h"
+
+namespace RobotConstants {
```

Comment:
were there not any PID constants anywhere? or are those going to get stored elsewhere?

Reply:
nevermind there's the Parameters file